vdm-io/Joomla-Component-Builder
# Improment to language constant replacment

Pull Request for Issue gh-198 .

### Summary of Changes
Updated the compiler to also target JText::script() & Joomla.JText._() in language constant replacement. Added field counter. 

### Testing Instructions
add some JText::script() & Joomla.JText._() to your component site or custom admin view (custom script) areas and it should be converted.

### Expected result
Converting JText::script() & Joomla.JText._()

### Actual result
There was no implementation for this before.

### Documentation Changes Required
We may need to add some notes around the custom script areas to inform users of this feature.
